### PR TITLE
Update one-time-password-sms.yaml

### PR DESCRIPTION
**Pull request by bigludo7**
Add E164 pattern for Phone number (as in other API)

### CODE REVIEW

File: code/API_definitions/one-time-password-sms.yaml [hunk 1]
```diff
@@ -179,6 +179,7 @@ components:
     PhoneNumber:
       description: A phone number belonging to the user. 'E164 with +' format
```
Comment (monamok) [inline code/API_definitions/one-time-password-sms.yaml:180]: We internally usually use: "^\+\d{1,15}"
But if we want '+'  to be optional we should update the description as well.

```suggestion
      description: A phone number belonging to the user. E.164 format (starting with country code). Optionally prefixed with '+'.
```

**Review by monamok — COMMENTED** [review]
If we change it, should we change the version too?

**Review by bigludo7 — COMMENTED** [review]
updated version

**Comment by bigludo7** [comment]
> 

I've took the pattern from the QoD API which is for me the flagship. I'm not against your proposal but we need to apply it on all the API featuring this msidn/mobileNumber.

**Comment by monamok** [comment]
> > 
> 
> I've took the pattern from the QoD API which is for me the flagship. I'm not against your proposal but we need to apply it on all the API featuring this msidn/mobileNumber.

I'm ok with the change you're proposing. I menat that if we change it, this pattern says that '+' is optional but in the description we're saying that it's not. So we should also align the description saying that '+' is optional.

**Comment by bigludo7** [comment]
Oh sorry I did get it !
Yes you're right !
I will update the yaml.

**Review by bigludo7 — COMMENTED** [review]
Add comment in phone number description to indicate '+' as optional

**Review by monamok — APPROVED** [review]
Thanks.

File: code/API_definitions/one-time-password-sms.yaml [hunk 2]
```diff
@@ -177,8 +177,9 @@ components:
         - authentication_id
         - code
     PhoneNumber:
-      description: A phone number belonging to the user. 'E164 with +' format
+      description: A phone number belonging to the user. 'E164 with an optional +' format
```
Comment (DT-DawidWroblewski) [inline code/API_definitions/one-time-password-sms.yaml:180]: looks good

**Review by DT-DawidWroblewski — APPROVED** [review]

**Pull request merged**